pandas/core/indexes/base.py

```diff
@@ -3718,9 +3719,14 @@ def get_indexer(
 
             indexer = self._engine.get_indexer(target.codes)
             if self.hasnans and target.hasnans:
+                # After _maybe_cast_listlike_indexer, target elements which do not
```

Comment:
can you put a "# GH#????" here pointing back to the original issue

Reply:
Done